cli/commands/prepare.ts

```diff
@@ -10,20 +10,24 @@ export const builder = () => {};
 
 export const handler = () => {
   for (const network of networks) {
-    console.log(`parsing network ${network}`);
-    const manifest = getManifest(
-      getTheGraphChainName(network),
-      getRequestNetworkChainName(network),
-    );
-    if (!manifest) {
-      console.warn(`No contract found for ${network}`);
-    } else {
-      writeFile(`subgraph.${network}.yaml`, manifest);
-    }
+    writeManifest(network);
   }
 };
 
-const getTheGraphChainName = (chainName: string) => {
+export const writeManifest = (network: string) => {
+  console.log(`parsing network ${network}`);
+  const manifest = getManifest(
+    getTheGraphChainName(network),
+    getRequestNetworkChainName(network),
+  );
+  if (!manifest) {
+    console.warn(`No contract found for ${network}`);
+  } else {
+    writeFile(`subgraph.${network}.yaml`, manifest);
+  }
+}
```

Comment:
I moved this logic to an exported function so I could access it from `compare.ts`